streamtube.js

```diff
@@ -7,6 +7,11 @@ var streamToTube = function(stream, maxDivergence, minDistance) {
 	var points = stream.points;
 	var velocities = stream.velocities;
 	var divergences = stream.divergences;
+	var maxNorm = 0;
+
+	for (var i = 0; i < points.length; i++) {
+		maxNorm = Math.max(vec3.length(velocities[i]), maxNorm);
```

Comment:
This is a per-stream maxNorm. You probably want to compute a maxNorm of all the streams instead and pass it to streamToTube.